### DISCUSSION TITLE
Cant run examples/api

### DISCUSSION THREAD

**Discussion by DevMash** [tauri-apps/tauri#7695] [Q&A]
I followed the step from the readme but I get following error. Any ideas?
https://github.com/tauri-apps/tauri/blob/dev/examples/api/README.md

![tauri2](https://github.com/tauri-apps/tauri/assets/3641524/2db11023-70d5-4e54-b482-2fd77600911c)

Before that error, I had this error: "Cannot find module '@tauri-apps/cli-win32-x64-msvc'"
I resolved it by installing the missing module in the tauri root folder (npm i @tauri-apps/cli-win32-x64-msvc)

**Comment by FabianLars** [comment]
Hmm, that clip error is a bit confusing, it shouldn't complain about that if you followed the readme (unless ofc the readme or scripts are outdated) :thinking: (I'll look into that when I'm not limited to my phone anymore)

Anyway, since installing the cli/component seemed to work try installing it again but using `@next` as the version since this repo's dev branch contains tauri v2

**Comment by DevMash** [comment]
Thanks, for the help!

I uninstall &reinstalled with @next:
npm uninstall @tauri-apps/cli-win32-x64-msvc
npm i @tauri-apps/cli-win32-x64-msvc@next

This worked! But now I see this error.
![tuari-error](https://github.com/tauri-apps/tauri/assets/3641524/84aa2008-d8db-47cc-9650-4cae0305a3fd)

**Reply by DevMash** [reply]
I was on **dev** branch. I switched to **1.x** branch and now the example works as expected.
